addons/docs/src/frameworks/svelte/config.ts

```diff
@@ -1,8 +1,12 @@
+import { extractArgTypes } from './extractArgTypes';
```

Comment:
can we get a couple unit tests for this?

Reply:
That would be difficult to with Svelte, i have to pass the component class as an argument to extractArgTypes, i cannot access the __docgen prop without instancing the class and i cannot inject the json documentation without the custom webpack loader.

i could refactor and create a separate function to unit test, like 

```ts
function docgenToArgTypes(docgen){
....
return results;
}
```